src/openforms/validations/api/views.py

```diff
@@ -29,7 +29,11 @@ class ValidatorsListView(ListMixin, APIView):
     serializer_class = ValidationPluginSerializer
 
     def get_objects(self):
-        return list(register.iter_enabled_plugins())
+        param_component = self.request.query_params.get("component", None)
+        filtered = filter(
+            lambda x: x.component == param_component, register.iter_enabled_plugins()
+        )
+        return [item for item in filtered]
```

Comment:
Maybe more readable? :smile: 

```suggestion
        return [plugin for plugin in register.iter_enabled_plugins() if plugin.component == param_component]
```
But also:
The query string should not be compulsory, if you don't specify it, I would expect that all the validators are returned.